src/bosh_azure_cpi/lib/cloud/azure/azure_client2.rb

```diff
@@ -1926,16 +1900,14 @@ def parse_network_interface(result, recursive: true)
         properties = result['properties']
         interface[:provisioning_state] = properties['provisioningState']
 
-        unless properties['enableIPForwarding'].nil?
-          interface[:enable_ip_forwarding] = properties['enableIPForwarding']
-        end
+        interface[:enable_ip_forwarding] = properties['enableIPForwarding'] unless properties['enableIPForwarding'].nil?
 
         unless properties['networkSecurityGroup'].nil?
-          if recursive
-            interface[:network_security_group] = get_network_security_group(properties['networkSecurityGroup']['id'])
-          else
-            interface[:network_security_group] = {:id => properties['networkSecurityGroup']['id']}
-          end
+          interface[:network_security_group] = if recursive
```

Comment:
at http://shopify.github.io/ruby-style-guide/, it says
# bad
result =
  if some_cond
    # ...
    # ...
    calc_something
  else
    calc_something_else
  end

# good
result = if some_cond
  # ...
  # ...
  calc_something
else
  calc_something_else
end

Reply:
looks like long indention is not need.

Reply:
autofixed by rubocop.